docs/getting-started/install_capi_operator.md

```diff
@@ -2,6 +2,71 @@
 sidebar_position: 3
 ---
 
-# Install CAPI Operator
+# Install Cluster API Operator
 
-This section needs to be filled out.
\ No newline at end of file
+This section describes how to install `Cluster API Operator` in the kubernetes cluster.
+
+## Installing CAPI and providers
+
+`CAPI` and desired `CAPI` providers could be installed using helm based installation for [`Cluster API Operator`](https://github.com/kubernetes-sigs/cluster-api-operator) or as a helm dependency for the `Rancher Turtles`.
+
+### Install as a `Rancher Turtles` dependency
+
+See the `Rancher Turtles` section for installing operator as a Helm [dependency](./install_turtles_operator.md#install-cluster-api-operator-as-a-helm-dependency)
+
+### Install with Helm
+To install `Cluster API Operator` with version `1.4.4` of the `CAPI` + `Docker` provider using helm, follow these steps:
+
+1. Add the Helm repository for the `Cluster API Operator` by running the following command:
+```bash
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
+```
+2. Update the Helm repository by running the following command:
+```bash
+helm repo update
+```
+3. Install the `Cluster API Operator` using the following command, which will also install `cert-manager`:
+```bash
+helm install capi-operator capi-operator/cluster-api-operator
+	--create-namespace -n capi-operator-system
+	--set infrastructure=docker:v1.4.4
+	--set cert-manager.enabled=true
+	--timeout 90s --wait # Core Cluster API with kubeadm bootstrap and control plane providers will also be installed
+```
+*Note: `cert-manager` is a hard requirement for CAPI and `Cluster API Operator`*
+
+To provide additional environment variables, choose some feature gates, or provide cloud credentials, similar to `clusterctl` [common provider](https://cluster-api.sigs.k8s.io/user/quick-start#initialization-for-common-providers), in `Cluster API Operator`, a variables secret could be used. A `name` and a `namespace` of the secret could be specified for the `Cluster API Operator`.
```

Comment:
We can also include that github token better be included in the secret, it's documented here https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/docs#installing-azure-infrastructure-provider